dataflow/src/org/checkerframework/dataflow/analysis/FlowExpressions.java

```diff
@@ -478,7 +480,9 @@ public boolean equals(Object obj) {
                 return false;
             }
             LocalVariable other = (LocalVariable) obj;
-            return other.element.equals(element);
+            return other.element.name.toString().equals(element.name.toString()) &&
```

Comment:
I think it's simpler to use `name.contentEquals` to compare Names - no need to call toString on both.